index.js

```diff
@@ -652,6 +665,78 @@ fastify.get("/api/addfile", async (request, reply) => {
     </body></html>`
 })
 
+fastify.get("/api/status", async (request, reply) => {
+  const { siteId } = request.query || ""
+  const token = idToken || ""
+  const email =
+    token !== "" ? JSON.parse(Buffer.from(token?.split(".")[1], "base64").toString()).email :
+    ""
+
+  if (!email || !siteId) {
+    reply.type("text/html")
+
+    return `
+    <html><body>
+      <h1>Error</h1>
+      <p>unknown site: "${siteId}" or user: "${email}"</p>
+      <h2>Actions</h2>
+      <ul>
+      <li><a href="/api/list">List sites</li>
+      </ul>
+    </body></html>`
+  }
+
+
+  const mqttClient = mqtt.connect(
+    `wss://iot.prod.xsitemanage.com/mqtt?token=${token}&contextType=ext-site&contextId=${siteId}`,
+    { clientId: `prod-user-${email}-${uuidv4().split('-')[0]}`, protocol: "wss" }
```

Comment:
```suggestion
    { clientId: `prod-user-${email}-${${Date.now()}`, protocol: "wss" }
```
Ehdotus omasta ui:sta. Poistaisin myös uuid-paketin kyydistä.